Move tentative global definitions to extern declarations to fix multiple
definition errors with GCC 10+ (-fno-common default). Provide the actual
storage in main.c.

--- camorama-0.20.7/src/camorama-globals.h.orig
+++ camorama-0.20.7/src/camorama-globals.h
@@ -29,10 +29,10 @@

 G_BEGIN_DECLS

-GtkWidget *prefswindow;
-int frames, frames2, seconds;
-GtkWidget *dentry, *entry2, *string_entry, *format_selection;
-GtkWidget *host_entry, *protocol, *rdir_entry, *filename_entry;
+extern GtkWidget *prefswindow;
+extern int frames, frames2, seconds;
+extern GtkWidget *dentry, *entry2, *string_entry, *format_selection;
+extern GtkWidget *host_entry, *protocol, *rdir_entry, *filename_entry;

 G_END_DECLS
 #endif                          /* !CAMORAMA_GLOBALS_H */
--- camorama-0.20.7/src/main.c.orig
+++ camorama-0.20.7/src/main.c
@@ -13,6 +13,11 @@
 #include <libv4l2.h>
 #include <stdlib.h>

+GtkWidget *prefswindow;
+int frames, frames2, seconds;
+GtkWidget *dentry, *entry2, *string_entry, *format_selection;
+GtkWidget *host_entry, *protocol, *rdir_entry, *filename_entry;
+
 static int ver = 0, max = 0, min;
 static int half = 0, use_read = 0, debug = 0;
 static gchar *video_dev = NULL;
